### PR TITLE
Migrate to hugo

### PR DESCRIPTION
**Pull request by noxilixon**
- Migrated the content from the old website to hugo
- Created automatic deployment via github actions
- Added apache configuration (similar to the setup used in utils.berlin.freifunk.net)

Currently available via [dev.ff.berlin/hugo](https://dev.ff.berlin/hugo)

### CODE REVIEW

**Comment by Akira25** [comment]
If we are going to host the website on a new vm, I'd love to go with nginx, as this is the server, we use throughout the rest of the ansible repo.

**Comment by noxilixon** [comment]
> If we are going to host the website on a new vm, I'd love to go with nginx, as this is the server, we use throughout the rest of the ansible repo.

Yes, i would like to have just the hugo stuff in this repo and make the server config via ansible. But this is out of the scope for this pr.

**Comment by noxilixon** [comment]
So I fixed some deployment stuff. I think this is ready to be merged. Looks like a good setup for me so that we can continue with the work on the content.

File: content/impressum.de.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+14167 Berlin
+
+**Vertreten durch den Vorstand**  
+Monic Meisel  
+Daniel Paufler  
+Andreas Bräu
+
+**Kontakt**  
+E-Mail: verein{at}foerderverein.freie-netzwerke.de  
+WWW: <http://foerderverein.freie-netzwerke.de>
```
Comment (Noki) [inline content/impressum.de.md:29]: http => https

File: layouts/shortcodes/img.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                    {{- end -}}
+                    {{- .Get "attr" | markdownify -}}
+                    {{- if .Get "attrlink" }}</a>{{ end }}</p>
+                {{- end }}
+            </figcaption>
+        {{- end }}
+    </figure>
+{{ else }}
+  {{ print "Alt text missing!" }}
+{{ end }}
```
Comment (Noki) [inline layouts/shortcodes/img.html:38]: I don't think this is a good idea.

File: static/logo.png
Comment (Noki) [inline static/logo.png:1]: an .svg file would be better here. do we have this as .svg?

File: www/static/berlin.json
Comment (Noki) [inline www/static/berlin.json:1]: This file is required for the https://api.freifunk.net/ and should not be deleted.

**Review by Noki — CHANGES_REQUESTED** [review]

**Comment by Noki** [comment]
I fixed quite some stuff. What's still open:
* Apache configuration fixes / changes
* Markdown linter fixes (probably add more custom short codes e.g. for iframes / and or replace linter with one that works better with Hugo)
* Change image shortcode

File: static/static/berlin.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    "number": 38,
+    "website": "https://wiki.freifunk.net/Berlin:Refugees"
+  },
+  "state": {
+    "focus": [
+      "infrastructure/backbone",
+      "Public Free Wifi",
+      "Free internet access"
+    ],
+        "lastchange":"2023-12-14T19:01:04+01:00",
```
Comment (noxilixon) [inline static/static/berlin.json:47]: This is dynamic data and should not be hardcoded like that. In the old setup this was:
```javascript
    "lastchange": "FILLED_IN_BY_MAKEFILE",
    "nodes": "FILLED_IN_BY_MAKEFILE"
```
We should also find a solution to update this data. Daily might be a good interval.

Comment (noxilixon) [reply static/static/berlin.json:47]: We should add this to the workflows and make it scheduled, so that this gets rebuild every day.

**Comment by noxilixon** [comment]
- [ ] Blank line in robots.txt before the Sitemap

File: static/static/berlin.json (hunk 3 above)
Comment (noxilixon) [reply static/static/berlin.json]: I moved the commands from the old makefile into the workflow. This is working, but I think a solution where hugo renders that file and gets some vars from the environment might be better.

Comment (Noki) [reply static/static/berlin.json]: I think we should just make this an issue to keep track of it and fix it later. In my opinion we can merge without this.

Comment (noxilixon) [reply static/static/berlin.json:47]: I removed the wip solution from the workflows. Implementing the better (hugo render based) solution is now an issue at #86.

**Review by Noki — APPROVED** [review]
Ready to be deployed!

**Pull request merged**